features/configuration_files/directory_specific_directives.feature

```diff
@@ -200,10 +200,7 @@ Feature: Directory directives
     Then the exit status indicates an error
     And stderr reports:
     """
-    You are trying to configure smell type IteratorsNested but we can't find one with that name.
-    Please make sure you spelled it right. (See 'docs/defaults.reek' in the Reek
-    repository for a list of all available smell types.)
-
+    Error: Bad configuration file: key 'IteratorsNested:' is undefined.
```

Comment:
This is somewhat less user-friendly. Perhaps we should enhance `ConfigFileError` to provide some guidance.

Reply:
Yes, looks good.